manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OBSProject.OBSStudio
+PackageVersion: 27.2.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x64
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/obsproject/obs-studio/releases/download/27.2.1/OBS-Studio-27.2.1-Full-Installer-x64.exe
+  InstallerSha256: E1277A2A7BA6252D2029667D70BC2E79CC9ED15E2823E6C1FFBF355DEBE98D63
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/obsproject/obs-studio/releases/download/27.2.1/OBS-Studio-27.2.1-Full-Installer-x86.exe
+  InstallerSha256: 32B33B75F3F7E69594ABDB4C01868ADCDE12C52BB9C30A6566AE165E85EE56D0
+ManifestType: installer
+ManifestVersion: 1.1.0
+ReleaseDate: 2022-02-14

```

Comment:
and the ReleaseDate field

Reply:
Done already corrected to same as release notes.